File: problemas/1100/1116.cs
using System;

namespace uri1115
{
    internal static class Program
    {
        private static void Main()
        {
            int n = int.Parse(Console.ReadLine());

            for (int i = 0; i < n; i++)
            {
                string[] entrada = Console.ReadLine().Split(' ');
                int x = int.Parse(entrada[0]);
                double y = double.Parse(entrada[1]);

                Console.WriteLine(y == 0 ? "divisao impossivel" : $"{x / y:F1}");
            }
        }
    }
}

File: problemas/1100/1116.dart
import 'dart:io';

void main() {
  final n = int.parse(stdin.readLineSync());

  for (var i = 0; i < n; i++) {
    final entrada = stdin.readLineSync().split(' ');
    final x = int.parse(entrada[0]);
    final y = int.parse(entrada[1]);

    stdout.writeln(y == 0 ? 'divisao impossivel' : (x / y).toStringAsFixed(1));
  }
}